config/rbac/role.yaml

```diff
@@ -4,9 +4,28 @@ kind: ClusterRole
 metadata:
   name: numaplane-role
 rules:
-- apiGroups:
-  - '*'
+- apiGroups: ["numaflow.numaproj.io"]
+  resources: ["*"]
+  verbs: ["*"]
+- apiGroups: ["numaplane.numaproj.io"]
+  resources: ["*"]
+  verbs: ["*"]
+- apiGroups: [""]
   resources:
-  - '*'
+    - configmaps
+    - secrets
+    - serviceaccounts
+    - namespaces
   verbs:
-  - '*'
+    - '*'
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources:
+    - rolebindings
+    - roles
+  verbs:
+    - '*'
+- apiGroups: ["apps"]
+  resources:
+    - deployments
+  verbs:
+    - '*'
```

Comment:
Oh, I guess `Deployment` needs to be here because of the Numaflow Controller, huh? Did you verify your list includes all of the types that are part of the Numaflow Controller manifest?

Reply:
yeah I verified. :)